src/chocolatey/infrastructure.app/nuget/NugetCommon.cs

```diff
@@ -49,6 +49,9 @@
 using Console = chocolatey.infrastructure.adapters.Console;
 using Environment = chocolatey.infrastructure.adapters.Environment;
 using System.Collections.Concurrent;
+using chocolatey.infrastructure.information;
+using chocolatey.infrastructure.registration;
+using chocolatey.infrastructure.app.services;

```

Comment:
Can these be added in the correct location?

Reply:
Fixed.